docs/demos-consumers-implementation-guidelines.md

```diff
@@ -0,0 +1,46 @@
+
+# Demos consumers implementation guidelines
+
+## wireframes
+
+Consumers MUST implement those wireframes :
```

Comment:
> Consumers MUST implement those wireframes : https://www.drupal.org/node/2818741#comment-12114776

What is your opinion if we change the sentence to `Consumers MUST implement the following wireframe: https://www.drupal.org/node/2818741#comment-12114776` 
or 
`Consumers MUST implement the wireframes found [here](https://www.drupal.org/node/2818741#comment-12114776)`